dev-python/pygresql/pygresql-5.0.4.ebuild

```diff
@@ -19,6 +19,8 @@ SLOT="0"
 KEYWORDS="~alpha amd64 ~hppa ~ia64 ~ppc ~sparc x86"
```

Comment:
Missing stable sparc.

Your tree is a bit out of date. sparc was stabled a short time ago.

Reply:
Sure, by me ;) This PR was created in a different branch while the stabilization waited for being picked up.